packages/hawtio/src/plugins/camel/contextsdash/ContextsDash.tsx

```diff
@@ -0,0 +1,345 @@
+import {
+  Button,
+  Card,
+  CardBody,
+  Dropdown,
+  DropdownItem,
+  KebabToggle,
+  Modal,
+  ModalVariant,
+  Text,
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem,
+} from '@patternfly/react-core'
+import { AsleepIcon, InfoCircleIcon, PlayIcon, Remove2Icon } from '@patternfly/react-icons'
+import { Table, TableBody, TableHeader, TableProps, wrappable } from '@patternfly/react-table'
+import { IResponse } from 'jolokia.js'
+import { AttributeValues } from '@hawtiosrc/plugins/connect/jolokia-service'
+import React, { useEffect, useState, useContext } from 'react'
+import { PluginNodeSelectionContext } from '@hawtiosrc/plugins'
+import { log } from '../globals'
+import { contextsService, ContextDashAttributes } from './contexts-service'
+import { eventService } from '@hawtiosrc/core'
+import { workspace } from '@hawtiosrc/plugins/shared'
+
+export const ContextsDash: React.FunctionComponent = () => {
+  const { selectedNode } = useContext(PluginNodeSelectionContext)
+  const [isReading, setIsReading] = useState(true)
+
+  const emptyCtxs: ContextDashAttributes[] = []
+  const [contexts, setContexts] = useState(emptyCtxs)
+  const [selectedCtxId, setSelectedCtxId] = useState<string[]>([])
+  const [isOpen, setIsOpen] = useState(false)
+  const [isConfirmDeleteOpen, setIsConfirmDeleteOpen] = useState(false)
+
+  const onSelectContext = (ctx: ContextDashAttributes, isSelecting: boolean) => {
+    const otherSelectedCtx = selectedCtxId.filter(c => c !== ctx.context)
+    setSelectedCtxId(isSelecting ? [...otherSelectedCtx, ctx.context] : [...otherSelectedCtx])
+  }
+
+  const selectAllContexts = (isSelecting = true) => {
+    setSelectedCtxId(isSelecting ? contexts.map(c => c.context) : [])
+  }
+
+  const isContextSelected = (ctx: ContextDashAttributes) => {
+    return selectedCtxId.includes(ctx.context)
+  }
+
+  const onDropdownToggle = (isOpen: boolean) => {
+    setIsOpen(isOpen)
+  }
+
+  const isStartEnabled = (): boolean => {
+    return selectedCtxId.some(id => {
+      const ctx = contexts.find(c => c.context === id)
+      return ctx && ctx.state === 'Suspended'
+    })
+  }
+
+  const onStartClicked = () => {
+    contexts
+      .filter(ctx => {
+        const id = selectedCtxId.find(id => ctx.context === id)
+        return id && ctx.state === 'Suspended'
+      })
+      .forEach(ctx => {
+        contextsService
+          .startContext(ctx)
+          .then(() => {
+            eventService.notify({
+              type: 'success',
+              message: 'Camel context started successfully',
+              duration: 3000,
```

Comment:
let's follow the same duration settings with the legacy version, unless there's specific reason not to do so. For these parts, I don't think we need to set special durations. Default should just work.